### DISCUSSION TITLE
How to join to an existing trace while knowing the traceId?

### DISCUSSION THREAD

**Discussion by Lingzhou314** [apache/skywalking#10766] [General]
For example, I saved the traceId in db and it will be executed by the scheduled job afterwards, How to add this job execution to this trace instead of a new one?

**Comment by wu-sheng** [comment]
You should keep a trace context snapshot(ContextManager#capture) in the database, rather than a trace ID.
Seata's plugin does things in this way.

**Reply by Lingzhou314** [reply]
> You should keep a trace context snapshot(ContextManager#capture) in the database, rather than a trace ID. Seata's plugin does things in this way.

i got it, thank you for your reply.
